docs/api/anonymization.md

```diff
@@ -0,0 +1,532 @@
+# Anonymizing a CSV dataset
+
+The anonymization algorithms require 4 input parameters:
+
+1. The dataset to be loaded. The current anonymization algorithms support CSV datasets
+2. A list of ColumnInformation objects. These objects contain information about the column: if the column is categorical or numerical, if it is quasi, sensitive or non-sensitive, the weight they contribute to the information loss and if they are considered as part of the linking attributes for re-identification risk calculation. The `Column Information` section includes further details
```

Comment:
How can a user determine which columns are quasi, sensitive, or non-sensitive? I think it might be good to refer to the part of the code that does that (or its description).